Cargo.toml

```diff
@@ -264,8 +264,8 @@ axum = { version = "0.6.6", default-features = false, features = [
   "query",
   "ws",
 ] }
-axum-extra = "0.4.2"
-axum-server = { version = "0.5.1", default-features = false, features = [
+axum-extra = { version = "0.9", features = ["typed-header"] }
+axum-server = { git = "https://github.com/bmwill/axum-server.git", rev = "f44323e271afdd1365fd0c8b0a4c0bbdf4956cb7", version = "0.6", default-features = false, features = [
```

Comment:
What's different in your fork, out of curiosity?

Reply:
updated rustls to match what this PR updates it to